### PR TITLE
Take customization into account when updating an Order's product quantity

### LINKED ISSUE #23265: BO - Order page - Property Customization->quantity is not valid - Updating the quantity of customized product NOK
#### Describe the bug

In the BO > Orders > Order details page > when we try to update the quantity of the customized product for the second time => it will be NOK

#### Steps to Reproduce

1. Go to FO > Add a customized product `demo_14` to the cart with quantity = 1 > process to checkout
2. BO > Orders > View last order created
3. Increase the quantity of the product to 3 => save
4. Increase the quantity again to 4 => save 
5. See the error: the quantity ordered is NOK, it is displayed `2` instead of `4` with errors in the console
6. If you try to update it again you have different errors 
image

PS: PrestaShop1769 => ok
**Screenshots**

https://drive.google.com/file/d/1cd4N5geO3hLc-wvU2webw11IujgnewNj/view?usp=sharing

#### Additional information

* PrestaShop version: 1770 / 1171 / 1772build1
* PHP version: 7.1

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | When editing quantities of a product in an order in the BO, the customization wasn't took into account leading to mismatches and wrong quantities. This PR fixes it
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23265
| How to test?      | Please see #23265
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23601)

### CODE REVIEW

**Review by matks — COMMENTED** [review]
@atomiix It would be awesome to have a Behat test that follows the bug steps, which fails with current `1.7.7.x` implementation, and see that your change fixes it?

**Review by Progi1984 — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @atomiix,

I found an issue following these steps:

1. Go to FO > Add a customized product demo_14 to the cart with quantity = 1 > process to checkout
2. BO > Orders > View last order created
3. Increase the quantity of the product to 2 => save
4. See the error, the total products tax included is NOK

image

Thank you!

File: admin-dev/themes/new-theme/js/pages/order/view/order-prices-refresher.js [hunk 1]
```diff
@@ -49,9 +49,11 @@ export default class OrderPricesRefresher {
     $.getJSON(this.router.generate('admin_orders_product_prices', {orderId})).then(productPricesList => {
       productPricesList.forEach(productPrices => {
         const orderProductTrId = OrderViewPageMap.productsTableRow(productPrices.orderDetailId);
-
+        const quantity = productPrices.quantity > 1
+          ? `<span class="badge badge-secondary rounded-circle">${productPrices.quantity}</span>`
+          : productPrices.quantity;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/pages/order/view/order-prices-refresher.js]: Use something like that to fill the span
```js
        const quantityContent = $('<span class="badge badge-secondary rounded-circle"></span>');
        quantityContent.text(productPrices.quantity)
```

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @atomiix,

It is ok :heavy_check_mark: 
https://drive.google.com/file/d/1X2sFGhZTCrcV97NvdcBXAjBrwN00zx74/view?usp=sharing

Thank you!

**Comment by Progi1984** [comment]
Thanks @atomiix & @khouloudbelguith

**Pull request merged**